src/components/shared/badges/Badges.jsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { Chip } from '@mui/material/';
+import { createTheme, ThemeProvider  } from '@mui/material/styles';
+
+
+const defaultTheme = createTheme();
+
+const theme = createTheme({
+	components: {
+		MuiChip: {
+			variants: [
+				{
+					props: { variant: 'accent'},
+					style: {
+						padding: 0,
+						background:defaultTheme.palette.normal.main,
+						color: defaultTheme.palette.primary.main,
+									}
+				}
+			]
+		}
+	}
```

Comment:
putem seta asta direct in fisierul de `theme.js`. 
E mai corect sa avem un singur context de theme in aplicatie, asa avem garantia ca nu interfereaza intre ele

Reply:
<img width="939" alt="image" src="https://user-images.githubusercontent.com/61699077/190870390-0864b6f8-e14b-47dc-8207-46944c86e227.png">

Reply:
mi se parea si mie mai normal normal sa mi fac personalizarile in theme.jsx
decat in new component variants